DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/task_sequences/README.md

```diff
@@ -0,0 +1,302 @@
+# `Task Sequences` Sample
```

Comment:
```suggestion
# `Task Sequences (Parallel Loops)` Sample
```

Please also rename the directory from `task_sequences` to `task_sequences_parallel_loops` to distinguish from `task_sequences_hardware_reuse` that Juny is working on.